### DISCUSSION TITLE
How can pre-prosses data in ST Table

### DISCUSSION THREAD

**Discussion by mouad-mahfoud** [ng-alain/ng-alain#2356] [Q&A]
i have this component
` <st
      [data]="data"
      [columns]="modalColumns"
      (change)="change($event)"
      [ps]="5"
      [total]="totalElement"
      [req]="{ params: params }"
      [res]=""
    ></st>`
how [res] work to prosses data ?

**Comment by cipchk** [comment]
Pls provide an expected result first, you can also read the [res](https://ng-alain.com/components/st/en?#API) document.
